kratos/geometries/brep_surface.h

```diff
@@ -91,6 +92,21 @@ class BrepSurface
         , mpNurbsSurface(pSurface)
         , mOuterLoopArray(BrepOuterLoopArray)
         , mInnerLoopArray(BrepInnerLoopArray)
+    {
+        mIsTrimmed = !(mOuterLoopArray.size() == 0 && mInnerLoopArray.size() == 0);
```

Comment:
if you can compute it I would not store it

Reply:
It is a flag to optimize the simulation. The computation of the flag is still better than nothing, however, it is valid only for very new cases.